frontend/src/static/css/_extra.css

```
@@ -3252,24 +3252,25 @@ body.dark_theme .playlist-videos-list .item .item-content h3 span {
 }
 
 /* #################################################################################################### */
 
 body.dark_theme .user-action-form-inner {
-	background-color: #0b3145;
+	background-color: var(--site-content-box-bg-color);
 }
 
 body.dark_theme .user-action-form-inner input,
 body.dark_theme .user-action-form-inner textarea {
 	border-width: 1px 1px 2px 1px;
```

Stylelint (17.11.1)

[error] Duplicate property "border-width" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)